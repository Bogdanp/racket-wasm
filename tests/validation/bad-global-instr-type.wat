(module
 (global i32 (i32.const 1) (drop) (i32.const 2)))