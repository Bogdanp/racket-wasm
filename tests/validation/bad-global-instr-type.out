(#f . "global instructions must be constant\n  at: (instr:drop)\n  within: (global (globaltype i32 #f) (vector (instr:i32.const 1) (instr:drop) (instr:i32.const 2)))")